lib/prawn-fillform.rb

```diff
@@ -297,7 +286,9 @@ def acroform_fields_for_page(page)
     end
 
     def fetch_field_attribute(data, page, field_name, attribute)
-      value = data[page][field_name].fetch(attribute) rescue nil
+      page_number = page.to_s.split('_').last.to_i + 1
+      page_key = "page_#{page_number}".to_sym
+      value = data[page_key][field_name].fetch(attribute) rescue nil
```

Comment:
what was the reason that made you parse the page name to access the value?
how is this related to drawing a check box?